src/tower_struggle/scenes/level_01.clj

```diff
@@ -20,11 +20,49 @@
     "img/big-background.png")
    (t/tetromino [300 800])])
 
+(defn tower-outline
+  [x y w h]
+  (let [tower-width w
+        tower-height h
+        coords {:tl  [x y]
+                :tr  [(+ x tower-width) y]
+                :bl  [x (+ y tower-height)]
+                :br  [(+ x tower-width) (+ y tower-height)]}
+        sides {:left   (concat (coords :tl) (coords :bl))
+               :right  (concat (coords :tr) (coords :br))
+               :top    (concat (coords :tl) (coords :tr))
+               :bottom (concat (coords :br) (coords :bl))}]
+    sides))
```

Comment:
Couple things here, neither game-breaking, but each improve clarity a little bit.

The coords map isn't ever used as a map, we just use it's values, so this can be replaced with the four variables `tl`, `tr`, `bl`, `br`.

The sides map is created, and then returned immediately, it's cleaner just to return the map itself instead of binding it to a variable first.

```suggestion
        tl [x y]
        tr [(+ x tower-width) y]
        bl [x (+ y tower-height)]
        br [(+ x tower-width) (+ y tower-height)]]
    {:left   (concat tl bl)
     :right  (concat tr br)
     :top    (concat tl tr)
     :bottom (concat br bl)}))
```